app/controllers/profiles/passwords_controller.rb

```diff
@@ -0,0 +1,32 @@
+class Profiles::PasswordsController < Devise::PasswordsController
+  # GET /resource/password/new
+  def new
+    super
```

Comment:
For this and all methods in this file.

I don't understand why you have these. If Profiles::PasswordsController inherits from Devise::PasswordsController, then isn't a method with only `super` the same as having no method at all?